### ISSUE TITLE
Backslash after git push raises a bug

### ISSUE THREAD

**Issue by RandyPearson** [git-for-windows/git#2717]
![image](https://user-images.githubusercontent.com/752899/85598012-b2106a00-b63a-11ea-918d-ad378ea671ba.png)

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?
64-bit
2.16.2.windows.1

```
$ git --version --build-options

** insert your machine's response here **
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?
Windows 10
```
$ cmd.exe /c ver

** insert your machine's response here **
```

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other
CMD
 - What commands did you run to trigger this issue? If you can provide a
git push\ <<-- note the accidental backslash

**Comment by landstander668** [comment]
For what it's worth, I can't reproduce this  behaviour using the current [2.27.0](https://github.com/git-for-windows/git/releases/tag/v2.27.0.windows.1) release.

```
C:\Users\adric\test> git version --build-options
git version 2.27.0.windows.1
cpu: x86_64
built from commit: 907ab1011dce9112700498e034b974ba60f8b407
sizeof-long: 4
sizeof-size_t: 8

C:\Users\adric\test> git push\
git: 'push\' is not a git command. See 'git --help'.

The most similar command is
        push
```
So your best bet is probably to upgrade from 2.16.2. This is probably a good idea regardless, as IIRC there have been some security-related updates since that point.

**Comment by rimrul** [comment]
That issue should be fixed since 2.17, please test with a version that is a little newer.
